x/ccv/provider/keeper/validator_set_update.go

```diff
@@ -229,5 +196,14 @@ func (k Keeper) FilterValidators(
 // GetLastBondedValidators iterates the last validator powers in the staking module
 // and returns the first MaxValidators many validators with the largest powers.
 func (k Keeper) GetLastBondedValidators(ctx sdk.Context) ([]stakingtypes.Validator, error) {
-	return ccv.GetLastBondedValidatorsUtil(ctx, k.stakingKeeper, k.Logger(ctx))
+	maxVals, err := k.stakingKeeper.MaxValidators(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return ccv.GetLastBondedValidatorsUtil(ctx, k.stakingKeeper, k.Logger(ctx), maxVals)
+}
+
+func (k Keeper) GetLastActiveBondedValidators(ctx sdk.Context) ([]stakingtypes.Validator, error) {
```

Comment:
Add a godoc comment here. I wonder if other names here would be easier to parse. Why not call this `GetLastProviderConsensusBondedValidators` or something similar to make the distinction clear between this method and `GetLastBondedValidators`?

Reply:
Good point, they were indeed a bit too close. aldo added a comment